dev/firely/docker-compose.mssqlserver.yaml

```
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+
+  firely:
+    depends_on:
+      - db
+    environment:
+      VONK_Repository: SQL
+      VONK_SqlDbOptions:AutoUpdateConnectionString: Initial Catalog=VonkData;Data
+        Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
+      VONK_SqlDbOptions:AutoUpdateDatabase: "true"
+      VONK_SqlDbOptions:ConnectionString: Initial Catalog=VonkData;Data Source=db,1433;User
+        ID=sa;Password=SQLServerStrong(!)Password*
+      VONK_SqlDbOptions:SchemaName: vonk
+
```

From the files in the `data/` dir after running firely (`vonkadmin.db`), it looks like the Admin DB is still using SQLite. I was able to get everything running on MSSQL following the docs
```suggestion
      # Admin DB configuration
      # https://docs.fire.ly/projects/Firely-Server/en/latest/maintenance/administration_api.html
      VONK_Administration:Repository: SQL
      VONK_Administration:SqlDbOptions:SchemaName: vonkadmin
      VONK_Administration:SqlDbOptions:ConnectionString: Initial Catalog=VonkAdmin;Data Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
      VONK_Administration:SqlDbOptions:AutoUpdateDatabase: "true"
      VONK_Administration:SqlDbOptions:AutoUpdateConnectionString: Initial Catalog=VonkAdmin;Data Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
```

---

I'd prefer to keep the administration config as-is since [this section in the docs](https://docs.fire.ly/projects/Firely-Server/en/latest/setting_up_firely_server/configuration/db_sqlite.html#prefer-sqlite-for-firely-server-administration) recommends SQLite for Firely Server >=0.7.1 and mentions that MSSQL, MongoDB and Memory storage will be deprecated in the future.